test/composite_quadratures.cxx

```diff
@@ -23,14 +26,8 @@ TEST_CASE( "Spherical Quadratures", "[sph-quad]" ) {
 
     REQUIRE( npts == nrad * nang );
 
-    double res = 0;
-    for( auto i = 0; i < npts; ++i ) {
-      const auto p = s.points()[i];
-      const auto rsq = p[0]*p[0] + p[1]*p[1] + p[2]*p[2];
-      res += s.weights()[i] * std::exp(-rsq);
-    }
-
-    CHECK( res == Catch::Approx( M_PI * std::sqrt(M_PI) ) );
+    const std::string msg = "MK(150) x LL(770) Spherical";
+    test_quadrature<RadialGaussian>(msg, s, std::pow(M_PI,1.5), 1e-10);
```

Comment:
This looks sketchy

Reply:
How so?

Reply:
Ah, its *wrong*... b/c $r^2$ is already in the weights, fixing

Reply:
Well the reason I was confused was that you're integrating a radial function... but I guess the test might make sense.

Are you also testing the integrals with atomic orbitals with nonzero angular momentum?

Reply:
No, but that should be a part of another PR